tests/test_chat.py

```diff
@@ -881,6 +896,128 @@ async def make_assertion(*_, **kwargs):
         monkeypatch.setattr(chat.get_bot(), "decline_chat_join_request", make_assertion)
         assert await chat.decline_join_request(user_id=42)
 
+    async def test_create_forum_topic(self, monkeypatch, chat):
+        async def make_assertion(*_, **kwargs):
+            return (
+                kwargs["chat_id"] == chat.id
+                and kwargs["name"] == "New Name"
+                and kwargs["icon_color"] == 0x6FB9F0
```

Comment:
I use this in `test_forum.py`, but there it's a constant rather than a magic number. Maybe we should add a constant for a color in `conftest.py` and import it both in `test_chat.py` and in `test_forum.py`?

Reply:
a bit too excessive for a test imo